backend/franky/service/stackoverflow/stackoverflow.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List
+from urllib.parse import urljoin
+
+import requests
+
+from model import UserData
+from service import Service
+
+
+class StackoverflowServiceException(Exception):
+    pass
+
+
+class StackOverflow(Service):
+
+    def __init__(self, rest_endpoint='https://api.stackexchange.com/2.2'):
+        self._rest_endpoint = rest_endpoint
+        self._service_name = 'stackoverflow'
+        self._user_url = 'users'
+        self._tags_url = 'tags'
+
+    def user(self, user_name: str) -> UserData:
+        user_id = self._get_user_id(user_name)
+        user_tags = self._get_user_tags(user_id)
+        user = UserData(login=user_id, name=user_name, tags=user_tags)
+        return user
+
+    def _get_user_id(self, user_name: str) -> str:
+        payload = {'inname': user_name}
+        response_data = self._call(self._user_url, payload)
+        data = response_data['items']
+
+        if len(data) > 1:
+            user_id = self._get_unique_user(data, user_name)
+        elif len(data) == 1:
```

Comment:
I suppose we should handle the `len(data) == 1` case the same way we handle `len(data) > 1`.